heron/statemgrs/tests/java/BUILD

```diff
@@ -12,6 +12,7 @@ spi_deps_files = [
     "//heron/common/src/java:basics-java", 
     "//heron/spi/src/java:common-spi-java", 
     "//heron/spi/src/java:statemgr-spi-java",
+    "//heron/spi/src/java:utils-spi-java",
```

Comment:
Let us not directly depend on utils-spi-java; it contains too many deep dependencies. 
Two options can be considered:
1. Add more specific building target in spi/src/java:BUILD
2. Re-org classes in utils, for instance, put SchedulerUtils into spi/src/java/scheduler or something similar.